controllers/v1/observationSubmissionsController.js

```diff
@@ -1131,5 +1131,67 @@ module.exports = class ObservationSubmissions extends Abstract {
     })
   }
 
+
+  
+  /**
+  * @api {post} /assessment/api/v1/observationSubmissions/list  List Observation Submissions
+  * @apiVersion 1.0.0
+  * @apiName List Observation Submissions
+  * @apiGroup Observation Submissions
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
```

Comment:
@deepakr123  where is the apiSampleRequest ?? And observationId,solutionId,entitiyId should be on params and should be the get requests.

Reply:
Resolved